src/libraries/System.Collections.Immutable/tests/default.rd.xml

```diff
@@ -23,5 +23,46 @@
         </Method>
       </Type>
     </Assembly>
+
+    <Assembly Name="xunit.assert">
+      <Type Name="Xunit.Sdk.AssertEqualityComparer`1[[System.Collections.Generic.HashSet`1[[System.Collections.Generic.KeyValuePair`2[[System.String, System.Private.CoreLib],[System.String, System.Private.CoreLib]],System.Private.CoreLib]],System.Private.CoreLib]]" Dynamic="Required All">
+        <Method Name="CompareTypedSets" Dynamic="Required All">
+          <GenericArgument Name="System.Collections.Generic.KeyValuePair`2[[System.String,System.Private.CoreLib],[System.String,System.Private.CoreLib]],System.Private.CoreLib" />
+        </Method>
+      </Type>
+    </Assembly>
+
+    <Assembly Name="xunit.assert">
+      <Type Name="Xunit.Sdk.AssertEqualityComparer`1[[System.Collections.Generic.HashSet`1[[System.Collections.Generic.KeyValuePair`2[[System.UInt64, System.Private.CoreLib],[System.UInt64, System.Private.CoreLib]],System.Private.CoreLib]],System.Private.CoreLib]]" Dynamic="Required All">
+        <Method Name="CompareTypedSets" Dynamic="Required All">
+          <GenericArgument Name="System.Collections.Generic.KeyValuePair`2[[System.UInt64,System.Private.CoreLib],[System.UInt64,System.Private.CoreLib]],System.Private.CoreLib" />
+        </Method>
+      </Type>
+    </Assembly>
+
+    <Assembly Name="xunit.assert">
+      <Type Name="Xunit.Sdk.AssertEqualityComparer`1[[System.Collections.Generic.HashSet`1[[System.Collections.Generic.KeyValuePair`2[[System.Int32, System.Private.CoreLib],[System.Int32, System.Private.CoreLib]],System.Private.CoreLib]],System.Private.CoreLib]]" Dynamic="Required All">
+        <Method Name="CompareTypedSets" Dynamic="Required All">
+          <GenericArgument Name="System.Collections.Generic.KeyValuePair`2[[System.Int32,System.Private.CoreLib],[System.Int32,System.Private.CoreLib]],System.Private.CoreLib" />
+        </Method>
+      </Type>
+    </Assembly>
+
+    <Assembly Name="xunit.assert">
+      <Type Name="Xunit.Sdk.AssertEqualityComparer`1[[System.Collections.Generic.HashSet`1[[System.Int32,System.Private.CoreLib]],System.Private.CoreLib]]" Dynamic="Required All">
+        <Method Name="CompareTypedSets" Dynamic="Required All">
+          <GenericArgument Name="System.Int32,System.Private.CoreLib" />
+        </Method>
+      </Type>
+    </Assembly>
+
+    <Assembly Name="xunit.assert">
+      <Type Name="Xunit.Sdk.AssertEqualityComparer`1[[System.Collections.Generic.HashSet`1[[System.UInt64,System.Private.CoreLib]],System.Private.CoreLib]]" Dynamic="Required All">
+        <Method Name="CompareTypedSets" Dynamic="Required All">
+          <GenericArgument Name="System.UInt64,System.Private.CoreLib" />
+        </Method>
+      </Type>
+    </Assembly>
```

Comment:
Nit: we can avoid opening and closing the `Assembly` tag to make things more compact.

```suggestion
    <Assembly Name="xunit.assert">
      <Type Name="Xunit.Sdk.AssertEqualityComparer`1[[System.Collections.Generic.HashSet`1[[System.Collections.Generic.KeyValuePair`2[[System.String, System.Private.CoreLib],[System.String, System.Private.CoreLib]],System.Private.CoreLib]],System.Private.CoreLib]]" Dynamic="Required All">
        <Method Name="CompareTypedSets" Dynamic="Required All">
          <GenericArgument Name="System.Collections.Generic.KeyValuePair`2[[System.String,System.Private.CoreLib],[System.String,System.Private.CoreLib]],System.Private.CoreLib" />
        </Method>
      </Type>

      <Type Name="Xunit.Sdk.AssertEqualityComparer`1[[System.Collections.Generic.HashSet`1[[System.Collections.Generic.KeyValuePair`2[[System.UInt64, System.Private.CoreLib],[System.UInt64, System.Private.CoreLib]],System.Private.CoreLib]],System.Private.CoreLib]]" Dynamic="Required All">
        <Method Name="CompareTypedSets" Dynamic="Required All">
          <GenericArgument Name="System.Collections.Generic.KeyValuePair`2[[System.UInt64,System.Private.CoreLib],[System.UInt64,System.Private.CoreLib]],System.Private.CoreLib" />
        </Method>
      </Type>

      <Type Name="Xunit.Sdk.AssertEqualityComparer`1[[System.Collections.Generic.HashSet`1[[System.Collections.Generic.KeyValuePair`2[[System.Int32, System.Private.CoreLib],[System.Int32, System.Private.CoreLib]],System.Private.CoreLib]],System.Private.CoreLib]]" Dynamic="Required All">
        <Method Name="CompareTypedSets" Dynamic="Required All">
          <GenericArgument Name="System.Collections.Generic.KeyValuePair`2[[System.Int32,System.Private.CoreLib],[System.Int32,System.Private.CoreLib]],System.Private.CoreLib" />
        </Method>
      </Type>

      <Type Name="Xunit.Sdk.AssertEqualityComparer`1[[System.Collections.Generic.HashSet`1[[System.Int32,System.Private.CoreLib]],System.Private.CoreLib]]" Dynamic="Required All">
        <Method Name="CompareTypedSets" Dynamic="Required All">
          <GenericArgument Name="System.Int32,System.Private.CoreLib" />
        </Method>
      </Type>

      <Type Name="Xunit.Sdk.AssertEqualityComparer`1[[System.Collections.Generic.HashSet`1[[System.UInt64,System.Private.CoreLib]],System.Private.CoreLib]]" Dynamic="Required All">
        <Method Name="CompareTypedSets" Dynamic="Required All">
          <GenericArgument Name="System.UInt64,System.Private.CoreLib" />
        </Method>
      </Type>
    </Assembly>
```